planning/technology/storage/README.md

```diff
@@ -66,8 +66,47 @@ This section details the requirements and options specifically for the manuscrip
 
 ##### Potential Partners
 
-1. S3 - storage for git
-2. ElasticSearch - duplicate data to make searchable
+Jack is thinking that really we probably want all of these:
+
+1. git (for versioning)
+2. S3 (for durability)
+3. ElasticSearch (for real-time querying)
+
+If we treat S3 files as immutable, they serve as our disaster-recovery solution
+and easy rollback/restore source. Git repos are considered current 'source of
+truth' versions, unless there's a catastrophe and we manually fall back to an
+s3 snapshot. Elasticsearch is considered ephemeral, and gets completely blown
+away on every update (or we store things with git SHAs, which allows us to
+search by version, which could be cool). The git SHAs would be useful all over
+the place for keeping track of state. If everything's tagged with the sha, we
+don't have to worry so much about race conditions, etc. Could be massively
+overkill.
+
+One option would be to run our own git server (fairly trivial, though we'd
```

Comment:
I think if we use git, we'll have to run our own git server and figure out how to have all of that data stored on S3 or EC2 boxes with a bunch of storage. Perhaps that's done via an NFS file system? Would have to research it since I'm not very knowledgable on the subject. S3 is the obvious solution for backups.